core/store/orm/schema_test.go

```diff
@@ -0,0 +1,177 @@
+		"ORMMaxIdleConns":                           "ORM_MAX_IDLE_CONNS",
+		"ORMMaxOpenConns":                           "ORM_MAX_OPEN_CONNS",
+		"OperatorContractAddress":                   "OPERATOR_CONTRACT_ADDRESS",
+		"OptimismGasFees":                           "OPTIMISM_GAS_FEES",
+		"P2PAnnounceIP":                             "P2P_ANNOUNCE_IP",
+		"P2PAnnouncePort":                           "P2P_ANNOUNCE_PORT",
+		"P2PBootstrapPeers":                         "P2P_BOOTSTRAP_PEERS",
+		"P2PDHTAnnouncementCounterUserPrefix":       "P2P_DHT_ANNOUNCEMENT_COUNTER_USER_PREFIX",
+		"P2PListenIP":                               "P2P_LISTEN_IP",
+		"P2PListenPort":                             "P2P_LISTEN_PORT",
+		"P2PPeerID":                                 "P2P_PEER_ID",
+		"P2PPeerstoreWriteInterval":                 "P2P_PEERSTORE_WRITE_INTERVAL",
+		"Port":                                      "CHAINLINK_PORT",
+		"ReaperExpiration":                          "REAPER_EXPIRATION",
+		"ReplayFromBlock":                           "REPLAY_FROM_BLOCK",
+		"RootDir":                                   "ROOT",
+		"SecureCookies":                             "SECURE_COOKIES",
+		"SessionTimeout":                            "SESSION_TIMEOUT",
+		"StatsPusherLogging":                        "STATS_PUSHER_LOGGING",
+		"TLSCertPath":                               "TLS_CERT_PATH",
+		"TLSHost":                                   "CHAINLINK_TLS_HOST",
+		"TLSKeyPath":                                "TLS_KEY_PATH",
+		"TLSPort":                                   "CHAINLINK_TLS_PORT",
+		"TLSRedirect":                               "CHAINLINK_TLS_REDIRECT",
+		"TriggerFallbackDBPollInterval":             "TRIGGER_FALLBACK_DB_POLL_INTERVAL",
+		"UnAuthenticatedRateLimit":                  "UNAUTHENTICATED_RATE_LIMIT",
+		"UnAuthenticatedRateLimitPeriod":            "UNAUTHENTICATED_RATE_LIMIT_PERIOD",
+	}
+
+	schemaT := reflect.TypeOf(ConfigSchema{})
+	for i := 0; i < schemaT.NumField(); i++ {
+		field := schemaT.FieldByIndex([]int{i})
+		item, found := items[field.Name]
+
+		//
+		// ╭──╮   ╭────────────────────────────────────╮
+		// │  │   │ It looks like you're trying to add │
+		// @  @  ╭│ configuration variable!            │
+		// ││ ││ ││                                    │
+		// ││ ││ ╯╰────────────────────────────────────╯
+		// │╰─╯│
+		// ╰───╯
+		//
+		// If this test is failing, you've probably added a new configuration
+		// variable, please make sure to:
+		//
+		// 1. Update the changelog
+		// 2. Update the ConfigPrinter found in core/store/presenters/presenters.go
+		//    if you think this variable needs to be shown in the UI
+		// 3. Update the documentation page
+		// 4. Add your new config variable to this test
+		//
+
+		require.True(t, found, fmt.Sprintf("New test variable: '%s', see test comment for guide on steps to follow when adding a configuration variable", field.Name))
```

Comment:
Should put Clippy in the test failure method :grin:

Reply:
lol might do that in a follow up